.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Question
+about: Ask A Question About flowR
+title: 'Question:'
+labels: enhancement
```

Comment:
Maybe not this label but another one? We have https://github.com/Code-Inspect/flowr/labels/question for that.